skyportal/facility_apis/observation_plan.py

```diff
@@ -385,6 +385,10 @@ def generate_plan(
                 [request.payload["exposure_time"]]
                 * len(request.payload["filters"].split(","))
             ),
+            # avoid the galactic plane?
+            'doAvoidGalacticPlane': requests[0].payload.get("galactic_plane", False),
+            # galactic latitude to exclude
+            'galactic_limit': request.payload["galactic_latitude"],
```

Comment:
I'm not a fan of the mixed use of camel case and underscores.

Reply:
Yeah all inputs are underscores, unfortunately, gwemopt is camel case so we are stuck with that :)